example/lib/widgets/drawer/floating_menu_button.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class FloatingMenuButton extends StatelessWidget {
+  const FloatingMenuButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return PositionedDirectional(
+      start: 16,
+      top: 16,
+      child: SafeArea(
+        child: Container(
+          decoration: BoxDecoration(
+            color: Theme.of(context).colorScheme.background,
+            borderRadius: BorderRadius.circular(999),
+          ),
+          padding: const EdgeInsets.all(8),
+          child: Row(
+            children: [
+              Builder(
```

Comment:
I don't think that `Builder` is needed here, the `context` of `FloatingMenuButton` should already have access to the Scaffold.

Reply:
Good point, fixed.